overdrive.py

```diff
@@ -164,6 +207,17 @@ def __init__(self, _db, collection):
         # This is set by an access to .collection_token
         self._collection_token = None
 
+    def endpoint(self, url, **kwargs):
+        """Create the URL to an Overdrive API endpoint.
+
+        :param url: A template for the URL.
+        :param kwargs: Arguments to be interpolated into the template.
+           The server hostname will be interpolated automatically; you
+           don't have to pass it in.
+        """
+        kwargs.update(self.hosts)
```

Comment:
Would this be considered a kind of merge between the two objects but the result going into `kwargs`?

Reply:
It is, but **kwargs is a new dict every time, so it's fine to modify it, and `self.hosts` won't be modified.